eng/scripts/Publish-MavenPackages.ps1

```diff
@@ -247,20 +247,17 @@ foreach ($packageDetail in $packageDetails) {
 
       if ($LASTEXITCODE -eq 0) {
         Write-Information "Package $($packageDetail.FullyQualifiedName) deployed"
-        break
+        exit 0
       }
 
-      Write-Information "Release attempt $attemt exited with code $LASTEXITCODE"
+      Write-Information "Release attempt $attempt exited with code $LASTEXITCODE"
```

Comment:
I don't know if it's an easy add-on here or not, but generally we've been trying to enable powershell strict mode in more places across our scripts.